### DISCUSSION TITLE
Transaction to other currency account is displayed as an Expense

### DISCUSSION THREAD

**Discussion by StoicaRemus** [firefly-iii/firefly-iii#9517] [Firefly III] [answered]
Hello, something is bothering me while doing transfers. I have set up multiple accounts, one of them in another currency.
When doing transfers from Account1(USD) to Account2(USD), everything is OK. 
When doing a transfer from Account1(USD) to Account3(Pounds), the transfer is displayed as an Expense, which messes up the report.
Am I not understanding correctly, shouldn't the transfer from A2>A3 not count as an expense as in the case of A1>A2?
Thanks

**Comment by JC5** [comment]
Hey, thanks for starting a discussion!

Are you talking about reports, by any chance?

**Reply by StoicaRemus** [reply]
Yes, the transaction is displayed in the Report section in the Expenses, does not show in the main Dashboard on the total of [Spent] or [[In + out this period]

**Reply by JC5** [reply, marked as answer]
If account A3 is not in the report, this is what happens, yes.

**Reply by StoicaRemus** [reply]
You are correct, I did not check the A3 account while generating the report. Thank you for pointing this out!
